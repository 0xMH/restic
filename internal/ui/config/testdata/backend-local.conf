password = "geheim"

backend "foo" {
    type = "local"
    path = "/srv/data/repo"
}

backend "bar" {
    path = "/srv/data/repo"
}
