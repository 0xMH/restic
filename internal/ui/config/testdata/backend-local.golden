{
  "Repo": "",
  "Password": "geheim",
  "PasswordFile": "",
  "Backends": {
    "bar": {
      "Type": "local",
      "User": "",
      "Host": "",
      "Path": "/srv/data/repo"
    },
    "foo": {
      "Type": "local",
      "User": "",
      "Host": "",
      "Path": "/srv/data/repo"
    }
  },
  "Backup": {
    "Target": null,
    "Excludes": null
  }
}
